packages/vscode-extension/src/constants.ts

```diff
@@ -3,3 +3,6 @@ export enum ConfigurationKey {
   ArmSupportEnabled = "armSupportEnabled",
   BicepEnvCheckerEnable = "validateBicep",
 }
+
+export const migrateV1DocUrl =
+  "https://github.com/OfficeDev/TeamsFx/blob/dev/docs/vscode-extension/migrate-v1/migrate-v1.md";
```

Comment:
Make this an aka link. (More flexible and we are able to get traffic report)

Reply:
Good point, let me create an aka link for it.